engines/llama/build.gradle

```diff
@@ -43,15 +44,15 @@ processResources {
         }
         copy {
             from jnilibDir
-            into "${project.projectDir}/build/classes/java/main/native/lib"
+            into "${baseResourcePath}/native/lib"
         }
 
         // write properties
-        def propFile = file("${project.projectDir}/build/classes/java/main/native/lib/llama.properties")
+        def propFile = file(baseResourcePath + "/native/lib/llama.properties")
         propFile.text = "version=${llamacpp_version}-${version}\n"
```

Comment:
Did you verify whether changes to the versions specified here will trigger task execution?

Reply:
Fixed - added input properties